distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -178,10 +202,20 @@ limit_data limit data = case limit of
         if limit <= extreme.length then extreme.take (..First limit) else
             extreme + data.take (..Sample (limit - extreme.length))
 
+
+## PRIVATE
+add_fields : Builder -> Integer -> Integer -> Vector | Nothing
```

Comment:
A bit of personal preference but IMHO the code is more readable as `Vector.Builder` - as then we know what kind of builder this is.